Test/WebTestCase.php

```diff
@@ -554,7 +555,7 @@ protected function getUrl($route, $params = array(), $absolute = false)
      *
      * @return void
      */
-    public function isSuccessful($response, $success = true, $type = 'text/html')
+    public function isSuccessful(Response $response, $success = true, $type = 'text/html')
```

Comment:
well, I don't know the history of that bundle (which I do really love by the way), but the ApiDoc tells that a response object is requried and this method uses internally the methods of the symfony response object so I cannot understand why there's a BC break.